src/octoprint/util/gcodeInterpreter.py

```diff
@@ -271,6 +280,8 @@ def _load(self, gcodeFile, printer_profile, throttle=None):
 	def _parseCuraProfileString(self, comment, prefix):
 		return {key: value for (key, value) in map(lambda x: x.split("=", 1), zlib.decompress(base64.b64decode(comment[len(prefix):])).split("\b"))}
 
+	def getWarning(self):
```

Comment:
A further complication is that start end gcode segments might move outside
the maximum print volume.

On 4 March 2016 at 13:19, Gina Häußge <notifications@github.com> wrote:

> In src/octoprint/util/gcodeInterpreter.py
> <https://github.com/foosel/OctoPrint/pull/1254#discussion_r55028146>:
>
> > @@ -271,6 +280,8 @@ def _load(self, gcodeFile, printer_profile, throttle=None):
> >  	def _parseCuraProfileString(self, comment, prefix):
> >  		return {key: value for (key, value) in map(lambda x: x.split("=", 1), zlib.decompress(base64.b64decode(comment[len(prefix):])).split("\b"))}
> >
> > +	def getWarning(self):
>
> This is not Java, we don't need getters for everything :) (Yes, I know,
> I've done that myself, I'm trying to get rid of such things while
> refactoring)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/foosel/OctoPrint/pull/1254/files#r55028146>.
>